tests/test_filter.py

```diff
@@ -19,122 +19,75 @@ def calendar_event():
-        """Test setting user agent"""
+    def test_filter_regex_exclude_but_regex_include(self) -> None:
         filt = Filter("['/um/']", "['/crip/']")
         assert filt.filter("summary", "description") is True
 
-    def test_filter_regex_exclude_ignore_case(self):
-        """Test setting user agent"""
+    def test_filter_regex_exclude_ignore_case(self) -> None:
         filt = Filter("['/UM/i']", "")
         assert filt.filter("summary", "description") is False
 
-    def test_filter_regex_exclude_ignore_case_multiline(self):
-        """Test setting user agent"""
+    def test_filter_regex_exclude_ignore_case_multiline(self) -> None:
         filt = Filter("['/CRiPT-$/im']", "")
-        assert (
-            filt.filter(
-                "summary",
-                """descript-
-ion""",
-            )
-            is False
-        )
-
-    def test_filter_regex_exclude_ignore_case_multiline_dotall(self):
-        """Test setting user agent"""
+        assert filt.filter("summary", """descript- ion""", ) is False
+
+    def test_filter_regex_exclude_ignore_case_multiline_dotall(self) -> None:
         filt = Filter("['/cript-.ion/sim']", "")
-        assert (
-            filt.filter(
-                "summary",
-                """descript-
-ion""",
-            )
-            is False
-        )
-
-    def test_filter_regex_exclude_multiline(self):
-        """Test setting user agent"""
+        assert filt.filter("summary", """descript- ion""", ) is False
+
+    def test_filter_regex_exclude_multiline(self) -> None:
         filt = Filter("['/cript-$/m']", "")
-        assert (
-            filt.filter(
-                "summary",
-                """descript-
-ion""",
-            )
-            is False
-        )
-
-    def test_filter_regex_exclude_multiline_dotall(self):
-        """Test setting user agent"""
+        assert filt.filter("summary", """descript- ion""", ) is False
+
+    def test_filter_regex_exclude_multiline_dotall(self) -> None:
         filt = Filter("['/cript-.ion/ms']", "")
-        assert (
-            filt.filter(
-                "summary",
-                """descript-
-ion""",
-            )
-            is False
-        )
-
-    def test_filter_regex_exclude_dotall(self):
-        """Test setting user agent"""
+        assert filt.filter("summary", """descript- ion""", ) is False
+
+    def test_filter_regex_exclude_dotall(self) -> None:
         filt = Filter("['/cript-./s']", "")
-        assert (
-            filt.filter(
-                "summary",
-                """descript-
-ion""",
-            )
-            is False
-        )
+        assert filt.filter("summary", """descript- ion""", ) is False
+
+    def test_filter_with_description_none(self, calendar_event: CalendarEvent) -> None:
+        calendar_event.description = None
+        filt = Filter("['exclude']", "['include']")
+        assert filt.filter_event(calendar_event) is True
```

Comment:
By makding the change above to the filter() method, this test should test filt.filter() and pass in explicit summary and description values.